### DISCUSSION TITLE
Blog: Inconsistencies with Documentation

### DISCUSSION THREAD

**Discussion by matt328** [squidfunk/mkdocs-material#6106] [Q&A] [answered]
I followed exactly what the first portion of the docs say as far as setting up a blog, but for some reason I always get an extra 'Blog' title in the navigation:

![image](https://github.com/squidfunk/mkdocs-material/assets/706207/621f80b2-5b1a-484c-a714-529126bad3bd)

I'm using v9.4.2.  My project is located here https://github.com/matt328/portfolio if additional details are needed. Thanks for having a look!

**Comment by kamilkrzyskow** [comment, marked as answer]
Looks to me like you didn't enable index pages for sections: https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#section-index-pages

**Reply by matt328** [reply]
That did it! Thanks so much.
